scripts/ready_yet/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+pathlib
```

Comment:
pathlib has been part of the standard library since Python 3.4: https://docs.python.org/3/library/pathlib.html.

I don't think you want to install this dependency, as it hasn't been updated since 2014: https://pypi.org/project/pathlib/.

Reply:
Thanks for catching this! https://github.com/getsentry/sentry-python/pull/3863